src/webaudio.js

```diff
@@ -145,6 +145,10 @@ export default class WebAudio extends util.Observer {
         this.state = null;
         /** @private */
         this.explicitDuration = params.duration;
+        /** @private */
+        this.dest = null;
```

Comment:
i don't like `dest` as a var name on a class level. Can you call it something like `sinkDestination` or `sinkStreamDestination`? Same goes for `this.audio`, something like `sinkAudioElement`?